docs: Add docs for GenSX Cloud (#589)
gensx-inc/gensx: website/docs/src/content/quickstart.mdx

![blog writing trace](/quickstart/blog-trace.png)

> ## Running the dev server

Reviewer: Not worth doing now, but at some point we should probably convert this to a multipage quickstart. It's very overwhelming in it's current form. Doesn't feel so "quick" with all the new additions.

Author: Agree we should do this. At a certain point we might want to just focus the quickstart on doing an immediate deployment. 

Opened https://github.com/gensx-inc/gensx/issues/597 to track a future update.

> GenSX provides a local development with local REST APIs that match the shape of workflows deployed to GenSX Cloud. Run the dev server from the CLI:

Reviewer: ```suggestion
GenSX provides a local development server with local REST APIs that match the shape of workflows deployed to GenSX Cloud. You can run the dev server from the CLI:
```

Reviewer: We should also add in a sentence before this like "Now that you've built your workflows, you can easily turn them into REST APIs."

Keeps the tutorial/quickstart type vibe going.

```bash
# Start the development server
gensx start src/workflows.tsx
```

The development server provides several key features:

- **Hot reloading**: Changes to your code are automatically detected and recompiled
- **API endpoints**: Each workflow is exposed as a REST endpoint
- **Swagger UI**: Interactive documentation for your workflows at `http://localhost:1337/swagger-ui`
- **Local storage**: Built-in support for blob storage and databases

You'll see something like this when you start the server:

```bash
🔍 Starting GenSX Dev Server...
ℹ Using project name from gensx.yaml: my-project
ℹ Starting development server...
✔ Compilation completed
✔ Generating schema

📋 Available workflows:
- ResearchAndWriteWorkflow: http://localhost:1337/local/my-project/default/workflows/ResearchAndWriteWorkflow/run

✅ Server is running. Press Ctrl+C to stop.
```

You can now test your workflow by sending requests to the provided URL using any HTTP client, or using the built-in Swagger UI at `http://localhost:1337/swagger-ui`.

## Deploying your project to GenSX Cloud

> GenSX Cloud provides serverless deployment with zero configuration:

Reviewer: ```suggestion
Now that you've tested your APIS locally, you can deploy them to the cloud. GenSX Cloud provides serverless deployment with zero configuration:
```

```bash
# Deploy your project to GenSX Cloud
gensx deploy src/workflows.tsx -ev OPENAI_API_KEY
```

This command:

1. Builds your TypeScript code for production
2. Bundles all dependencies
3. Uploads the package to GenSX Cloud
4. Creates REST API endpoints for each workflow
5. Configures serverless infrastructure

For production deployments, you can target a specific environment:

```bash
# Deploy to production environment
gensx deploy src/index.ts --environment production
```

> ### Running a workflow from the CLI

Reviewer: Leading with the CLI feels strange here. I think a better path for this quickstart would be
a) go test it in the console
b) copy the code and go run it

I think we should show sample code for calling a workflow somewhere here because ultimately we want people calling workflows programatically

Reviewer: Not even sure we need to mention the CLI here (less is more)

Once deployed, you can execute your workflows directly from the command line:

```bash
# Run a workflow synchronously
gensx run ResearchAndWriteWorkflow --input '{"topic":"quantum computing"}'

# Save the output to a file
gensx run ResearchAndWriteWorkflow --input '{"topic":"quantum computing"}' --wait --output result.json
```

The CLI makes it easy to test your workflows and integrate them into scripts or automation.

### Running a workflow from the GenSX console

The GenSX Cloud console provides a visual interface for managing and executing your workflows:

1. Log in to [app.gensx.com](https://app.gensx.com)
2. Navigate to your project and environment
3. Select the workflow you want to run
4. Click the "Run" button and enter your input
5. View the results directly in the console

![Running a workflow in the console](/cloud/console-playground.png)

The console also provides API documentation and code snippets for your workflows as well as execution history and tracing for all previous workflow runs.

## Adding persistence with storage

> GenSX Cloud provides built-in storage solutions with zero configuration:

Reviewer: ```suggestion
Now that you've deployed your first workflow, you can add in storage so you can build more sophisticated workflows. GenSX Cloud provides built-in storage solutions with no setup required:
```

Reviewer: Probably also worth quickly stating "GenSX offers three types of storage: blob storage, sql database storage and, full text and vector search storage."
